### DISCUSSION TITLE
MiraiAnnouncements.members() 方法只能获取到50个看过群公告的人 不能全部获取

### DISCUSSION THREAD

**Discussion by yujumpjump** [simple-robot/Discussions#87] [打扰了]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

在使用MiraiAnnouncements.members()这个方法我想获取所有看过群公告的人 一共有300个人看过  但是他只返回了50个人 不能获取全部的人

**Comment by ForteScarlet** [comment]
与预期不符、疑似bug等内容请反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml)
